airbyte-workers/src/main/java/io/airbyte/workers/helper/ConnectionHelper.java

```diff
@@ -89,8 +92,15 @@ public static StandardSync updateConnectionObject(final WorkspaceHelper workspac
           .withTimeUnit(update.getSchedule().getTimeUnit())
           .withUnits(update.getSchedule().getUnits());
       newConnection.withManual(false).withSchedule(newSchedule);
+      // Also write into the new field. This one will be consumed if populated.
+      newConnection
```

Comment:
thinking out loud: 

there is some duplication here and the connection helpers which I'm slightly worried about since they aren't located close to each other. I don't think this is a concern as we should be removing this relatively quicker. wanted to verbalise my thoughts.